docs/quickstart.md

```diff
@@ -109,6 +109,57 @@ Notice that the save mode is now `Append`. In general, always use append mode un
 [Querying](#query) the data again will now show updated trips. Each write operation generates a new [commit](http://hudi.incubator.apache.org/concepts.html) 
 denoted by the timestamp. Look for changes in `_hoodie_commit_time`, `rider`, `driver` fields for the same `_hoodie_record_key`s in previous commit. 
 
+## Delete data {#deletes}
+Delete records for the HoodieKeys passed in. Lets first generate a new batch of insert and delete the same. Query to verify
+that all records are deleted.
+
+```
+val inserts = convertToStringList(dataGen.generateInserts(10))
+val df = spark.read.json(spark.sparkContext.parallelize(inserts, 2))
+df.write.format("org.apache.hudi").
+    options(getQuickstartWriteConfigs).
+    option(PRECOMBINE_FIELD_OPT_KEY, "ts").
+    option(RECORDKEY_FIELD_OPT_KEY, "uuid").
+    option(PARTITIONPATH_FIELD_OPT_KEY, "partitionpath").
+    option(TABLE_NAME, tableName).
+    mode(Overwrite).
+    save(basePath);
+
+// Fetch the rider value for the batch of records inserted just now
+val roDeleteViewDF = spark.
+    read.
+    format("org.apache.hudi").
+    load(basePath + "/*/*/*/*")
+roDeleteViewDF.registerTempTable("hudi_ro_table")
+spark.sql("select distinct rider from  hudi_ro_table where").show()
+
+// replace the rider value in below query to a value from above. "rider-213" is first batch and "rider-284" is second batch.
+val ds = spark.sql("select uuid, partitionPath from hudi_ro_table where rider = 'rider-284'")
+
+// issue deletes
```

Comment:
ideally,. we just have this part and remove everything above this line, to keep the quickstart small

Reply:
I am deleting an entire batch of inserts and hence thought will do a new batch of inserts and delete the entire batch.

Reply:
So, if I do the same with initial insert batch, then all records will be deleted. But don't want to disrupt the flow for rest of the quick start.

Reply:
I can move it as the last section. Hope thats fine.

Reply:
lets just delete a few existing records? and show that.. you can use `.limit(2)` to say get just 2 records out of the existing table and delete it

Reply:
Lets have it after incremental query.. deletes will conclude the flow of writing and reading nicely